sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp

```diff
@@ -66,13 +66,6 @@ namespace Azure { namespace Storage { namespace Blobs {
   } // namespace Models
 
   class StartCopyBlobOperation : public Azure::Core::Operation<Models::BlobProperties> {
-  public:
-    Azure::ETag ETag;
-    Azure::DateTime LastModified;
-    std::string CopyId;
-    Models::CopyStatus CopyStatus;
-    Azure::Nullable<std::string> VersionId;
```

Comment:
CL entry needed.

Reply:
Operation was added in this same release. So we don't use CL entry.

Reply:
I am looking at the previous beta.8 release, it was there.
https://github.com/Azure/azure-sdk-for-cpp/commit/572208baaccdc22da16c6e5d7de6aa52cb12d528

https://github.com/Azure/azure-sdk-for-cpp/blob/572208baaccdc22da16c6e5d7de6aa52cb12d528/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp#L71-L78

Same with the one below, but it was called Result instead of Operation. It's a breaking change.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/blob/572208baaccdc22da16c6e5d7de6aa52cb12d528/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_client.hpp#L211

In beta.8 release, `StartCopy` returned `Responst<>`. In beta.9 release, we changed it to `Operation<T>` and added CL.

> Changed the return type of StartCopyFromUri and StartCopyIncremental API from a Response<T> to the particular Operation<T> type called StartBlobCopyOperation directly.

https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/storage/azure-storage-blobs/CHANGELOG.md#breaking-changes

Reply:
I get all that. In beta.8 the response had fields like ETag/LastModified.

In beta.9, not only did we change it to `operation<t>` (which you mentioned in the changelog), we are also removing these public fields. That's not mentioned in the changelog.

The user, in beta.8 could get the `Response<T>` and access these fields. That would no longer work, hence recommended the CL entry **also** about the field removal. **Up to you though.**